xcube_server/app.py

```diff
@@ -56,6 +56,7 @@ def new_application(name: str=DEFAULT_NAME):
         (prefix + url_pattern('/datasets'), GetDatasetsJsonHandler),
         (prefix + url_pattern('/variables/{{ds_name}}'), GetVariablesJsonHandler),
         (prefix + url_pattern('/coords/{{ds_name}}/{{dim_name}}'), GetCoordinatesJsonHandler),
+        (prefix + url_pattern('/{{ds_name}}/{{var_name}}/legend.png'), GetLegendHandler),
```

Comment:
URL style contradicts the one of other URLs in this API. Please be consistent. I suggest naming the resource first

```suggestion
        (prefix + url_pattern('/legend/{{ds_name}}/{{var_name}}.png'), GetLegendHandler),
```